languages/java/exercises/concept/conditionals-ternary/.docs/after.md

```diff
@@ -0,0 +1,33 @@
+_ternary operators_ can be thought of as being _if-else_ inline. It is a good example that most of the times you have different ways of achieving the same result.
```

Comment:
The `after.md` document should contain all information that is introduced in the .docs/introduction.md document. See [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd). It doesn't necessarily have to use the exact same wording, but it feels like this document is missing the introduction bit.

Reply:
Oh, i didn't remember that, i will modify it

Reply:
Great!